apps/costcontrol/elements/datausage-tab.html

```diff
@@ -16,20 +16,20 @@
 
         <ul class="toggle-data" data-type="list">
           <li id="mobileItem">
-            <gaia-checkbox id="mobileCheck">
-              <label>
-                <span data-l10n-id="mobile-usage">Mobile usage</span>
-                <span class="overview" id="mobileOverview"></span>
-              </label>
-            </gaia-checkbox>
+            <label class="pack-checkbox">
```

Comment:
I was talking with Wilson and he advised to switch back to BB for now and entirely remove obsoleted ```shared/elements/gaia_checkbox```. New gaia-components/gaia-checkbox (that can be used via bower) also doesn't support RTL yet and cost control doesn't use bower currently - so I decided to make it in the easiest way. Does it work for you?